chroma_core/management/commands/print-settings.py

```diff
@@ -16,6 +17,21 @@ class Command(BaseCommand):
 
     help = """Prints out selected settings in env variable format"""
 
+    def _load_config(self, config_file):
```

Comment:
https://pypi.org/project/python-dotenv/ ?

Reply:
I'd rather not bring in any extra dependencies if we can avoid it.